api/Attr.json

```diff
@@ -151,7 +151,7 @@
               "notes": "This API was previously available on the <a href='https://developer.mozilla.org/docs/Web/API/Node'><code>Node</code></a> API."
             },
             "edge": {
-              "version_added": true
+              "version_added": "13"
```

Comment:
This is wrong, `document.createAttribute('x').prefix` is null (not undefined) in Edge 12, which means it was supported then. It's an old API so that makes sense.